user_guide/data_management_view.rst

```diff
@@ -0,0 +1,173 @@
+
+|
+
+If there are more than four actions available for the record, they will be collected under the 
+*"{Entity Name} Actions*" drop-down.
+
+|
+
+.. image:: ./img/data_management/view/view_action_buttons_2.png
+
+|
+
+Use Contact Details on a View Page
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Contact details related to a record are displayed as links. 
+
+You can click the phone number link to call it via Hangouts or log a call, start writing an email with a click on the 
+address, initiate Skype session directly from the View page.
+
+Add a Tag to the Record
+^^^^^^^^^^^^^^^^^^^^^^^
+Tags are non-hierarchical keywords assigned to a record to provide additional information, simplify search and filter 
+records. Tags can be added to a record from its View page, as described in the :ref:`Tags <user-guide-tags-add>` guide.
+
+Share the Details with Other Users
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Sharing records is very convenient when you need assistance from other system users who might have no access to the 
+related record. 
+
+For example, there is a task related to an opportunity that should be performed by a person from a 
+marketing team. Marketing associates don't have access to opportunity records, but the sales manager can share the 
+record with a specific user (or group of users), whose assistance is needed.
+
+To share a record, click the :guilabel:`Share`, and enter the name of the user to share the record with in the 
+*"Share with"* field or click the list icon to select such user(s). 
+
+|
+
+.. image:: ./img/data_management/view/view_share_01.png
+
+|
+
+The selected users will gain access to the record.
+
+Sometimes, there also appears a need to exchange information between organizations. For example, a customer has moved 
+countries, and sales reps of one office want to share the information to the second office.
+Such sharing is possible, but only from a system organization.
+
+(This means that a user with system-wide access should log-in to the 
+:ref:`System organization <user-ee-multi-org-system>`), access this record and share it with users in other 
+organizations.
+
+Get to Other Pages
+^^^^^^^^^^^^^^^^^^
+
+- You can get to the View page of the previous, the next, the first and the last records of the grid with the paginator 
+  in the top right corner of the page:  
+
+ 
+|
+
+.. image:: ./img/data_management/view/view_paginator.png
+
+|  
+
+- If there is a related entity, its identifier will be a link to the View page of this entity. 
+
+- You can get to the corresponding grid of records with the link in the top left corner of the page:
+
+|
+
+.. image:: ./img/data_management/grid/grid_from_view.png
+
+|
+
+.. hint::
+
+    If you have reached a View page or Create/Edit form from a grid, and now click  link to get back to the grid, this 
+    grid will be look in the same as it did when you left it for the View page (the same filters and order applied). 
```

Comment:
Edit: If you have reached a View page or Create/Edit form from a grid, and click on a link to get back to the grid, it will look the same as it did when you left it (the same filters and order applied).